create-icons.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
True, but I'd rather have a separate script than try to mush the entire process into a single shell line. I find it much more pleasant to read and edit.

Reply:
> True, but I'd rather have a separate script than try to mush the entire process into a single shell line. I find it much more pleasant to read and edit.

Personally, I do not see anything wrong with it, but I've been collecting lashes for this approach by Flathub maintainers.
You can leave it as it is, but it may lead to delay in accepting this request. I'm just warning you.

Reply:
It can be multiline thanks to `>` or just correct indentation.

"collecting lashes" certainly is correct approach to feedback you received, @scx.